Tests/StreamChatUITests/SnapshotVariant.swift

```diff
@@ -10,6 +10,10 @@ enum SnapshotVariant: String, Hashable, CaseIterable {
     case large
     case extraExtraExtraLarge
     
+    static var userInterfaceStyleCases: [SnapshotVariant] {
+        [small, large]
+    }
```

Comment:
@VojtaStavik @DominikBucher12 I still think this is not optimal, because the screenshots will print "small" and "large", so it will be difficult to understand which one is dark, and which one is light unless we override the `rawValue: String` of the `SnapshotVariant`. I still think the `OptionSet` is a bit more flexible, and probably a better solution for our needs. WDYT?

Reply:
I vote for the `OptionSet` to have this clear and flexible. Also I like rebasing my pull requests :D

Reply:
-> Slack